twyla/kubedeploy/__init__.py

```diff
@@ -1,40 +1,19 @@
-import json
 import os
 import shutil
 import sys
 import tempfile
 
 import click
-import colorama
-import kubernetes.client
-import pip
-import yaml
-
 import git
-
-from .kube import Kube, KubeException
-from . import docker_helpers
-
-
-# Use a colorized prompt to differentiate output of this script from output
-# that is generated by called programs and libraries
-colorama.init(autoreset=True)
-PROMPT = '>> '
-
-
-def prompt(msg: str, indent: int=0):
-    indentation = ' ' * indent
-    sys.stdout.write(colorama.Fore.GREEN + PROMPT + indentation)
-    print(msg)
-
-
-def error_prompt(msg: str):
-    sys.stdout.write(colorama.Fore.RED + PROMPT)
-    print(msg)
-
+import pip
+from twyla.kubedeploy import docker_helpers
+from twyla.kubedeploy.kube import Kube
+from twyla.kubedeploy.prompt import error_prompt, prompt
 
 
 def download_requirements(force: bool=False):
+    if not os.path.isfile('requirements.txt'):
```

Comment:
A hacky way to support non-Python services.

Reply:
It's a better idea to check for `setup.py`, as `requirements.txt` is actually superseded by pipfile.

Reply:
Generally I agree. This whole function should be different if we want to be somewhat project agnostic. For now I would like to leave the guard in as is because the download works with this exact file.